third_party/move/move-vm/runtime/src/runtime_type_checks.rs

```diff
@@ -285,6 +285,10 @@ impl RuntimeTypeCheck for FullRuntimeTypeCheck {
             Instruction::Abort => {
                 operand_stack.pop_ty()?;
             },
+            Instruction::AbortMsg => {
+                operand_stack.pop_ty()?;
```

Comment:
We should check types are expected (u64 and a reference to a vector of u8) - not sure why it has not been don for abort instruction above (that one needs to check the type is u64)